src/styles/mixins.scss

```diff
@@ -4,4 +4,11 @@
   font-weight: 400;
   font-size: 14px;
   color: $primary-dark-text;
+}
+
+@mixin container {
```

Comment:
About container. We can use it more flexibly by giving it the necessary properties in media expressions to support different screen sizes. Just a suggestion, I don't know the design features.